app/scripts/app.module.js

```diff
@@ -35,7 +35,8 @@
       'mwl.calendar',
       'app.offline',
       'app.administration',
-      'app.openmrsRestServices'
+      'app.openmrsRestServices',
+      'notification'
```

Comment:
rename the module to 'app.notification' instead of 'notification' to avoid future conflicts with external libraries